source/material_model/rheology/elasticity.cc

```diff
@@ -273,78 +316,168 @@ namespace aspect
       template <int dim>
       void
       Elasticity<dim>::fill_reaction_outputs (const MaterialModel::MaterialModelInputs<dim> &in,
-                                              const std::vector<double> &average_elastic_shear_moduli,
+                                              const std::vector<double> &,
                                               MaterialModel::MaterialModelOutputs<dim> &out) const
       {
-        if (in.current_cell.state() == IteratorState::valid && this->get_timestep_number() > 0 && in.requests_property(MaterialProperties::reaction_terms))
+        // TODO: also evaluate at t = 0?
```

Comment:
I initially thought we would want to do this at t=0, but now I'm not sure it makes sense to do so if we have not actually computed a numerical time step yet? However, I think we would still definitely want to add the force term to the RHS of the momentum equation at t=0.

Reply:
At the moment, we use the fixed_elastic_timestep when no numerical timestep is available yet, so that's probably fine. If the initial fields are zero, the reaction terms will be zero. Thinking about it again, I think the reaction terms should be computed at t0 just in case the initial stresses on the fields are not zero.

The force terms are already added at t0.